### PR TITLE
multi exist as bucket level interface and more tests

### PR DESCRIPTION
**Pull request by grosser**
@lukebakken 

FYI pretty_print prints the same as inspect, so not sure why it even needs to be defined ...

### CODE REVIEW

File: spec/riak/bucket_spec.rb [hunk 1]
```diff
@@ -178,8 +180,19 @@
 
       @results = @bucket.get_many %w{key1 key2}
 
-      expect(@results['key1']).to eq(@object1)
-      expect(@results['key2']).to eq(@object2)
+      expect(@results).to eq('key1' => @object1, 'key2' => @object2)
+    end
+  end
```
Comment (grosser) [inline spec/riak/bucket_spec.rb]: making this test catch when unexpected keys were added

**Comment by grosser** [comment]
updated to fix rubocop, should be all green now

**Comment by grosser** [comment]
How about a merge ? :)

**Comment by lukebakken** [comment]
Thanks, included in #309

**Pull request closed without merging**